src/modules/playRoom/components/PickItem.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Modal, Input } from 'antd';
+
+const PickItem = ({ img }) => {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >                
+        <input type="image" className="img-select" src={img.src} alt="" name="" value={img.value} />
```

Comment:
Do not leave attributes ```alt```, ```name``` empty, you can remove the attributes if not necessary